opsdroid/cli/config.py

```diff
@@ -125,3 +126,31 @@ def list_modules(ctx):
 
     """
     list_all_modules(ctx, ctx.obj, "config")
+
+
+@config.command()
+@click.pass_context
+@click.option(
+    "-d",
+    "verbose",
```

Comment:
I feel like this flag should either be `--debug` or `--verbose`. Mixing the two is a little strange. I'd also like to reserve `-d` in case there is something better to do with it.

I also agree this should be available on all commands.

Reply:
I've decided to go with `--verbose` here  because it seems a bit more logical since the user will get all the logs and know what's happening. If this flag is not set then the only thing shown to the user will be:

```
Opsdroid will build modules from config.
No databases in configuration. This will cause skills which store things in memory to lose data when opsdroid is restarted.
Opsdroid modules successfully built from config.
```

I'm not sure if it's worth to add a try/except and only show the successful message if nothing wrong happened 🤔

Reply:
> I'm not sure if it's worth to add a try/except and only show the successful message if nothing wrong happened 🤔

Yes please!

Reply:
We are handling exceptions pretty well so when we try to run the subcommand no exception is raised since we are handling it elsewhere. So the solution I found was to suppress any exception - this will make the success message only show when things go right. 

When things go wrong opsdroid will just show the appropriate log message telling that there was a problem. As an example, if the config fails the validation no exception is raised, the success message doesn't show and the validation error message is shown in the terminal.